.github/workflows/docs.yaml

```diff
@@ -1,33 +1,51 @@
-name: Build and Deploy Documentation
+name: Documentation
```

Comment:
How and where can we view this built documentation?
Is work required at the RTD website to setup hooks to pull the artifact?

Reply:
@aerorahul You can partially view it in GitHub.  For example, see this [link](https://github.com/jprestop/workflow-tools/blob/feature/UW-65/docs/index.rst).  However, I know that is not really what you want to see.  I'll reply to your review comment below with more information on this.